model/src/main/java/uk/gov/hmcts/fees2/register/data/model/FeeVersion.java

```diff
@@ -76,19 +79,22 @@ public BigDecimal calculateFee(BigDecimal amount) {
     @Override
     public String toString() {
         return "FeeVersion{" +
-            "fee=" + fee.getCode() +
+            "fee=" + fee +
             ", amount=" + amount +
             ", description='" + description + '\'' +
             ", version=" + version +
             ", status=" + status +
+            ", validFrom=" + validFrom +
+            ", validTo=" + validTo +
+            ", author='" + author + '\'' +
+            ", approvedBy='" + approvedBy + '\'' +
             ", memoLine='" + memoLine + '\'' +
-            ", directionType=" + directionType +
             ", feeOrderName='" + feeOrderName + '\'' +
             ", naturalAccountCode='" + naturalAccountCode + '\'' +
             ", statutoryInstrument='" + statutoryInstrument + '\'' +
             ", siRefId='" + siRefId + '\'' +
-            ", validFrom=" + validFrom +
-            ", validTo=" + validTo +
+            ", reasonForUpdate='" + reasonForUpdate + '\'' +
+            ", directionType=" + directionType +
```

Comment:
Is there any requirement to change the toString() implementation ?

Reply:
since added new field, updated toString() implementation accordingly

Reply:
I auto generated the toString() after adding reasonForUpdate, if we don't need directionType, I can remove it. That's anyway not relevant to the current functionality. Please suggest ....

Reply:
there might be some restrictions on Data Masking for PI data, so I would suggest to append only the new attribute

Reply:
Done